hypertrace-trace-enricher/helm/values.yaml

```diff
@@ -65,13 +65,32 @@ deploymentSelectorMatchLabels:
 ###########
 traceEnricherConfig:
   name: hypertrace-trace-enricher-config
-  kafka:
-    streams:
-      config:
-        metricsRecordingLevel: INFO
-        numStreamThreads: 2
-        bootstrapServers: "bootstrap:9092"
-        schemaRegistryUrl: "http://schema-registry-service:8081"
+  # Important kafka streams configurations which are used in config template goes here.
+  kafkaStreamsConfig:
+    bootstrapServers: "bootstrap:9092"
+    schemaRegistryUrl: "http://schema-registry-service:8081"
+    #  Core config
+    numStreamThreads: 2 # default = 1
+    commitIntervalMs: 30000 # default = 30000
+    # Common client (prodcuer, consumer, admin) configs
+    receiveBufferBytes: 4194304 # default = 32768 (kafka streams default)
+    sendBufferBytes: 4194304 # default = 131072 (kafka streams default)
+    # Producer configs
+    producerAcks: all # default: 1
+    producerBatchSize: 524288 # default = 16384
+    producerLingerMs: 1000 # default = 100 (kafka streams default)
+    producerCompressionType: "gzip" # default = none
+    producerMaxRequestSize: 10485760 # default = 1048576
+    producerBufferMemory: 134217728 # default = 33554432
+    # Consumer configs
+    consumerMaxPartitionFetchBytes: 4194304 # default = 1048576
+    consumerMaxPollRecords: 1000 # default = 1000 (kafka streams default)
+    consumerSessionTimeoutMs: 10000 # default = 10000
+    # Others
+    metricsRecordingLevel: INFO # default = INFO
```

Comment:
same comment as in https://github.com/hypertrace/hypertrace-ingester/pull/91/files#r549230957